src/features/content-answers/ContentAnswersModal.tsx

```diff
@@ -34,10 +34,11 @@ type Props = {
     currentUser?: User;
     file: BoxItem;
     isOpen: boolean;
+    onAsk: Function;
     onRequestClose: () => void;
```

Comment:
nit: do we want to be more specific than `Function` similar to `onRequestClose`?

Reply:
updated to change all of the callbacks in here